tck/src/main/java/org/eclipse/microprofile/config/tck/ConfigProviderTest.java

```diff
@@ -88,28 +88,18 @@ public static void setupCheck(){
 
     @Test
     public void testEnvironmentConfigSource() {
-        Map<String, String> env = System.getenv();
-        Properties properties = System.getProperties();
-        for (Map.Entry<String, String> envEntry : env.entrySet()) {
-            String key = envEntry.getKey();
-            if(!properties.containsKey(key)) {
-                String value = envEntry.getValue();
-                String cfgValue = config.getValue(key, String.class);
-                Assert.assertEquals(value, cfgValue, "The property "+key+" did not have the expected value.");
-            }
-        }
+        String value = System.getenv().get("MP_TCK_ENV_DUMMY");
```

Comment:
I suggest to also include some builtin environment variables, such as `PATH` etc to demonstrate not only the properties we added but also the builtin ones.